Test/BasicTest.cs

```diff
@@ -1,32 +1,33 @@
-using System;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System.Collections.Generic;
 using System.IO;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
 using MimeMapping;
 
 namespace Test
 {
     [TestClass]
     public class BasicTest
     {
-        Dictionary<string, string> _expectedTypes = new Dictionary<string, string> {
-            { "PNG", "image/png"},
-            { "png", "image/png"},
-            { "JPG", "image/jpeg"},
-            { "mp4", "video/mp4" },
-            { "exe", "application/x-msdownload" },
-            { "zip", "application/zip" },
-            { "torrent", "application/x-bittorrent" },
-            { "json", "application/json" },
-            { "asdfunknown", "application/octet-stream" }
+        private readonly Dictionary<string, string> _expectedTypes = new Dictionary<string, string>
+        {
+            {"PNG", "image/png"},
+            {"png", "image/png"},
+            {"JPG", "image/jpeg"},
+            {"mp4", "video/mp4"},
+            {"exe", "application/x-msdownload"},
+            {"zip", "application/zip"},
+            {"torrent", "application/x-bittorrent"},
+            {"json", "application/json"},
+            {"asdfunknown", "application/octet-stream"}
         };
 
         [TestMethod]
         public void TestCommonTypes()
         {
             foreach (var t in _expectedTypes)
             {
-                var foundType = MimeUtility.GetMimeMapping(t.Key);
+                var filePath = Path.Combine(Path.GetTempPath(), "examplefile." + t.Key);
+                var foundType = MimeUtility.GetMimeMapping(filePath);
```

Comment:
Nice catch here -- the `filePath` was never being used. Can you perform this fix within the `TestCommonTypesAsFilePaths` method? This `TestCommonTypes` is intended to test bare extension strings without the dot.